package.json

```diff
@@ -12,10 +12,14 @@
   },
   "dependencies": {
     "@apollo/client": "^3.7.1",
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
```

Comment:
won't this work without emotion? :( i'd generally encourage to use material icon variable font, because we're adding 3 unnecessary packages just for icons